Plots/Bar/NCL_bar_5.py

```diff
@@ -0,0 +1,248 @@
+# This produces the same results as NCL's yyyymm_to_yyyyfrac() function
+num_months = np.shape(date)[0]
+date_frac = np.empty_like(date)
+for n in np.arange(0, num_months, 1):
+    yyyy = int(date[n] / 100)
+    mon = (date[n] / 100 - yyyy) * 100
+    date_frac[n] = yyyy + (mon - 1) / 12
+
+###############################################################################
+# Plot:
+
+# Create two lists of values: one for years represented and another for Y values
+# in inches of the bottom left corner of year graph
+warm = [1951, 1953, 1957, 1963, 1965, 1969, 1972, 1976, 1982, 1987, 1991]
+heights_list = [
+    0.52, 0.764, 1.0, 1.25, 1.5135, 1.721, 2.0, 2.215, 2.5345, 2.72, 2.98
+]
+
+
+# A helper function to create a figure with multiple axes, each of which will have a different bar graph
+def create_figure(width, num_subplots, heights_list):
+    # Create global variables for the figures and an empty dictionary to store the axes
+    global fig, ax0, ax_dict
+    ax_dict = {}
+
+    # Create figure and default axis which will be visible
+    fig = plt.figure(figsize=(width + 2, (num_subplots / 4) + 2))
+    ax0 = fig.add_axes([0.5, 0.5, width, num_subplots / 4])
+
+    # Loop through heights in heights_list and create a different axis in ax_dict
+    # for each one
+    i = 1
+    for height in heights_list:
+        ax_dict["ax" + str(i)] = fig.add_axes(
+            [0.355, height, width + .295, 0.25 * .75])
+        i = i + 1
+
+
+# A helper function to create a bar chart for each axis
+def bar_plot(ax_name, year):
+    # Find year limits of each bar chart
+    year_start = year - 1
+    year_end = year + 3
+
+    # Find the indicies for each of the year limits
+    year_istart = int(np.where(np.round(date_frac, 3) == year_start)[0])
+    year_iend = int(np.where(np.round(date_frac, 3) == year_end)[0])
+
+    # Create each bar chart where it is red if it is above 0 and blue if below
+    ax_dict[ax_name].bar(date_frac[year_istart:year_iend],
+                         sst[year_istart:year_iend],
+                         align='edge',
+                         edgecolor='black',
+                         color=[
+                             'red' if (value > 0) else 'blue'
+                             for value in sst[year_istart:year_iend]
+                         ],
+                         width=.08,
+                         linewidth=2)
+    # Turn off axis so it is not visible
+    ax_dict[ax_name].axis("off")
+
+
+# A helper function to make the left y axis labels
+def left_tick_labels(num_plots):
+    tick_list = []
+    i = 0
+    tick_list.append("")
+    while i < num_plots:
+        tick_list.append("-1.0")
+        tick_list.append("0.0")
+        tick_list.append("1.0")
+        tick_list.append("")
+        i = i + 1
+    tick_list.append("")
+    return tick_list
+
+
+# A helper function to make the right y axis labels
+def right_tick_labels(num_plots, year_list):
```

Comment:
```suggestion
def right_tick_labels(year_list):
```

It doesn't look like you're not using the `num_plots` input to your helper function